src/middlewared/middlewared/migration/004_ntp_default_servers.py

```diff
@@ -0,0 +1,20 @@
+async def migrate(middleware):
+    if await middleware.call('system.product_type') != 'SCALE':
+        return
+
+    # There is a drawback with this approach, it will virtually not allow any user to use freebsd ntp server
```

Comment:
What do you mean, why?

Reply:
Ah, I thought that these migrations are applied on each upgrade but your previous comment explains that. This comment is no longer true, i'll make amends. Thank you.